src/constants.py

```diff
@@ -15,6 +15,9 @@
 LOG_DIR = PurePath("/var/log")
 LOG_FILE = LOG_DIR / "admin_ui.log"
 RULES_CONFIGMAP_FILE_NAME = "admin_ui_rules.json"
+OAUTH_SCOPES = "openid profile offline_access"
```

Comment:
question: should we add `email` also (even though we don't use the email claim yet) ?

Reply:
Perhaps I should remove `profile`?

Reply:
Profile is usually for first and last name at least, so we may not need it. Definitely not for now.
We could remove it now, or leave it and see if we can actually work with it. Up to you

Reply:
Left it for now